.gitignore

```diff
@@ -4,5 +4,6 @@
 /cloned/
 /output/
 /cache/
+/tests/fixture-code/cache/
```

Comment:
Unrelated cleanup, along with removing those files. These should never have been included in the repository.